Jenkinsfile

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env groovy
+
+pipeline {
+  agent {
+    label 'huge && bionic && x86_64 && !xenial && !trusty'
+  }
+
+  options {
+    timestamps()
+    timeout(time: 2, unit: 'HOURS')
+    quietPeriod(5)
+  }
+
+  environment {
+    NPROC = "0"
+  }
+
+  stages {
+    stage('Prepare') {
+      steps {
```

Comment:
could add a step here to abort the previous job if it still running:
```groovy
        script {
          def buildNumber = env.BUILD_NUMBER as int
          if (buildNumber > 1) milestone(buildNumber - 1)
          milestone(buildNumber)
       }
```

see also https://issues.jenkins-ci.org/browse/JENKINS-43353

Reply:
Thanks @tchaikov ! I already stumbled upon that trick (it looks really hacky... with the freestyle project it was so easy). I will add this one.

Reply:
Hope this https://github.com/jenkinsci/pipeline-milestone-step-plugin/pull/18 gets merged soon!